apps/wallet-mobile/src/components/StatusBar.tsx

```diff
@@ -12,6 +12,8 @@ export const StatusBar = ({type, overrideColor}: Props) => {
   const backgroundColor = type === 'dark' ? COLORS.WHITE : COLORS.BACKGROUND_BLUE
 
   useFocusEffect(() => {
+    if (Platform.OS === 'ios') return
```

Comment:
to remove the annoying warning

Reply:
Messages only appear on iOS because [setBackgroundColor](https://reactnative.dev/docs/statusbar#setbackgroundcolor-android) only works on Android.

I've done a small refactor and everything still works normally.